images/casperjs/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.fedoraproject.org/fedora:latest
+
+RUN cd /
+RUN yum -y update && yum install -y npm bzip2 fontconfig
+RUN npm config set user 0
+RUN npm config set unsafe-perm true
+RUN npm install -g phantomjs
+RUN npm install -g casperjs
+
+LABEL name="casperjs"
+LABEL version="0.1"
+LABEL maintaner="Chris Callegari"
+LABEL release-date="2020-10-05"
```

Comment:
Is this used for anything? I'm thinking whether it's worth the maintenance cost.

Reply:
Nope They're not used for anything.  I'm happy to remove the LABELs

Reply:
items removed